python/solutii/stefan_munteanu/paint/paint/cursor.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# *-* coding: UTF-8 *-*
+"""Tuxy dorește să împlementeze un nou paint pentru consolă.
+În timpul dezvoltării proiectului s-a izbit de o problemă
+pe care nu o poate rezolva singur și a apelat la ajutorul tău.
+Aplicația ține un istoric al tuturor mișcărilor pe care le-a
+făcut utlizatorul în fișierul istoric.tuxy
+Exemplu de istoric.tuxy:
+    STÂNGA 2
+    JOS 2
+    DREAPTA 5
+Fișierul de mai sus ne spune că utilizatorul a mutat cursorul
+2 căsuțe la stânga după care 2 căsuțe in jos iar ultima acțiune
+a fost să poziționeze cursorul cu 5 căsuțe în dreapta față de
+ultima poziție.
+El dorește un utilitar care să îi spună care este distanța dintre
+punctul de origine (0, 0) și poziția curentă a cursorului."""
+import math
+
+
+DIRECTII_VALIDE = ["SUS", "JOS", "STANGA", "DREAPTA"]
+
+
+def validare_inscructiune(instructiue):
+    """Testeaza pentru validitatea mutarii"""
+    if instructiue in DIRECTII_VALIDE:
```

Comment:
Poți returna direct `instructiune in DIRECTII_VALIDE`, nu mai ai nevoie de `if`.